src/empire/trade_prices.c

```diff
@@ -139,6 +139,28 @@ int trade_price_change(resource_type resource, int amount)
     return 1;
 }
 
+int trade_price_set_buy(resource_type resource, int new_price)
+{
+    if (new_price < 2) {
+        prices[resource].buy = 2;
```

Comment:
Why `2`? It seems you're just restricting the map maker creativity here.

Also, best use a `#define MIN_BUY_PRICE 2` instead of a magic number.

Reply:
> Why `2`? It seems you're just restricting the map maker creativity here.

I picked 2 since the sell price is limited to one.
Or do you mean I should be allowing negative values?

Reply:
Or do you mean allow for 0?

Reply:
Zero seems to much, but 1 buy/1 sell seems reasonable...

Reply:
Fixed. Min set to 1 for both buy and sell.